contracts/Governance/RadicleToken.sol

```diff
@@ -187,6 +187,35 @@ contract RadicleToken {
         return true;
     }
 
+    function burnFrom(address account, uint256 rawAmount) public virtual {
+        require(account != address(0), "RadicleToken::burnFrom: cannot burn from the zero address");
+        uint96 amount = safe96(rawAmount, "RadicleToken::approve: amount exceeds 96 bits");
+
+        address spender = msg.sender;
+        uint96 spenderAllowance = allowances[account][spender];
+        if (spender != account && spenderAllowance != uint96(-1)) {
+            uint96 newAllowance =
+                sub96(
+                    spenderAllowance,
+                    amount,
+                    "RadicleToken::burnFrom: burned amount exceeds spender allowance"
+                );
+            allowances[account][spender] = newAllowance;
+            emit Approval(account, spender, newAllowance);
+        }
+
+        balances[account] = sub96(
+            balances[account],
+            amount,
+            "RadicleToken::_burnTokens: burned amount exceeds balance"
+        );
+        emit Transfer(account, address(0), amount);
+
+        _moveDelegates(delegates[account], address(0), amount);
```

Comment:
I'm trying to wrap my head around this. Does this mean the user delegates to the zero address?

Reply:
Looking at `_moveDelegates`, I think this is correct :thinking:

Reply:
Conceptually yes, the voting power is burned along the tokens. In practice `_moveDelegates` checks for zero and doesn't apply anything to the zero address, only removes from the burner.